marketpulse/main/templates/fxosprice_view.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2 class="title">Firefox OS devices prices</h2>
+  <h3 class="title">your contributions</h3>
+
+  {% if contributions %}
+    <table class="table table-condensed">
+      <thead>
+        <tr>
+          <td>Device</td>
+          <td>When</td>
+          <td class="hidden-xs">Available</td>
+          <td>Shop</td>
+          <td></td>
+        </tr>
+      </thead>
+      <tbody>
+        {% for contribution in contributions %}
+          <tr>
+            <td>{{ contribution.device.name }}</td>
+            <td>{{ contribution.created_on|timesince }}</td>
+            <td class="hidden-xs">
+              {% if contribution.availability %}
+                <span class="text-success">yes</span>
+              {% else %}
+                <span class="text-danger">no</span>
+              {% endif %}
+            </td>
+            <td>{{ contribution.location.shop_name }}</td>
+            <td>
+              <a class="delete" href="{{ url('main:delete_fxosprice', contribution.pk) }}">
+                Delete
+              </a>
+            </td>
+          </tr>
+        {% endfor %}
+      </tbody>
+    </table>
+  {% else %}
```

Comment:
I don't think that we can check this ``else`` branch at the moment because of the ``ZeroDivisionError``. Looks good codewise, but please double check before merging.